src/test/java/org/broadinstitute/hellbender/tools/variantdb/nextgen/CreateVariantIngestFilesIntegrationTest.java

```diff
@@ -0,0 +1,56 @@
+package org.broadinstitute.hellbender.tools.variantdb.nextgen;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.testutils.ArgumentsBuilder;
+import org.broadinstitute.hellbender.testutils.IntegrationTestSpec;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.util.*;
+
+import static htsjdk.samtools.ValidationStringency.STRICT;
+
+public class CreateVariantIngestFilesIntegrationTest extends CommandLineProgramTest {
+
+    private static final String input_vcf_file = "NA12878.haplotypeCalls.reblocked.chr20.100k.vcf.gz";
+    private static final String interval_list_file = "wgs_calling_regions.hg38.chr20.100k.interval_list";
+    private static final String sample_map_file = "test_sample_map.tsv";
+    private static final File outputDir = createTempDir("output_dir");
+
+    // TODO why can't I run Collections.sort() here?
```

Comment:
You can't run anything other than variable definitions here (ie no blocks of code).  In Java you can run code here by saying:

```
static {
   // your code
}
```

but that's generally bad form since you don't really know when that code will run.  The better place to put this would be in the constructor for this class.  Although in your case I would just sort the three things by hand since they're literals anyway